### PR TITLE
Improve some editor strings for localization

### PR DESCRIPTION
**Pull request by timothyqiu**
* "Script" in locale editor means a writing system, i.e. the set of letters in which a language is written. It is put into its own context and a translator comment is added.
* "Up" and "Down" in network profiler mean the outgoing and incoming bandwidth. They are put into their own context and a translator comment is added.
* A newline is added to a warning in CollisionShape3D. This makes it consistent with its CollisionPolygon3D counterpart and other long warnings nearby.
* Unified several message strings that are similar (e.g. differ by only a space or colon).
* Use `TTRN()` for "Run N Instances".

### CODE REVIEW

File: editor/import/dynamic_font_import_settings.cpp [hunk 1]
```diff
@@ -1403,7 +1403,7 @@ DynamicFontImportSettings::DynamicFontImportSettings() {
 
 	label_glyphs = memnew(Label);
 	gl_hb->add_child(label_glyphs);
-	label_glyphs->set_text(TTR("Preloaded glyphs:") + " " + itos(0));
+	label_glyphs->set_text(vformat(TTR("Preloaded glyphs: %d"), 0));
```
Comment (akien-mga) [inline editor/import/dynamic_font_import_settings.cpp:1406]: Are there languages which would realistically need to move `%d` around here?
The `"Some text:" + " " + var` pattern is quite common throughout the codebase (I know the space after the `:` might not be the one you'd use in e.g. Chinese, but that's the most common pattern we use anyway).

**Review by akien-mga — APPROVED** [review]

File: editor/import/dynamic_font_import_settings.cpp (hunk 1 above)
Comment (timothyqiu) [reply editor/import/dynamic_font_import_settings.cpp:1406]: Including `%d` provides additional context. e.g. You know it's describing the number of glpyhs, not used as the label of a glphy list.
Weblate also complains about the translated string not including a trailing space but the original string has one.

Comment (akien-mga) [reply editor/import/dynamic_font_import_settings.cpp:1406]: Yeah we should never have trailing spaces in msgids.

File: editor/editor_locale_dialog.cpp [hunk 2]
```diff
@@ -446,7 +446,8 @@ EditorLocaleDialog::EditorLocaleDialog() {
 			vb_script_list->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 			{
 				Label *script_lbl = memnew(Label);
-				script_lbl->set_text(TTR("Script:"));
+				// TRANSLATORS: This is the label for a list of writing systems.
+				script_lbl->set_text(TTR("Script:", "Locale"));
```
Comment (bruvzg) [inline editor/editor_locale_dialog.cpp:450]: The same is relevant for the `TTR("Add Script")` in the `editor/plugins/font_config_plugin.cpp`.

File: editor/import/dynamic_font_import_settings.cpp (hunk 1 above)
Comment (bruvzg) [reply editor/import/dynamic_font_import_settings.cpp:1406]: We probably should use `vformat` everywhere, in some (rare) cases it's necessary to add control chars into the translation to ensure correct BiDi processing. Usually when it's something like `"Something something %s, %s and %s."` and inserted words are type names or something else that is not translated, but there might be other cases as well.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**